modules/standard/fun/ding_controller.py

```diff
@@ -0,0 +1,104 @@
+import random
+
+from core.command_param_types import SignedInt
+from core.decorators import command, instance
+from core.registry import Registry
+
+
+@instance()
+class DingController:
+    """
+        Port of Budabot fun module
+    """
+
+    def inject(self, registry: Registry):
+        self.command_alias_service = registry.get_instance("command_alias_service")
```

Comment:
since we aren't defining a default alias, this entire method is not needed anymore

Reply:
whoops! 😊  

Removed